cmd/login_cloud.go

```diff
@@ -1,50 +1,111 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 
+	"github.com/fatih/color"
+	"github.com/loadimpact/k6/stats/cloud"
 	"github.com/loadimpact/k6/ui"
-	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
+var (
+	token string
+	show  bool
+)
+
+func printToken(conf cloud.Config) {
+	label := "Token"
+	displayLabel := " " + color.New(color.Faint, color.FgCyan).Sprint("["+label+"]")
+	fmt.Fprintf(stdout, " "+displayLabel+": "+conf.Token+"\n")
```

Comment:
For consistency, just do

```go
fmt.Fprintf(stdout, " token: %s\n", ui.ValueColor.Sprint(conf.Token))`.
```

Honestly, you can probably restructure the whole function to just end in this without needing a function at all.